ReactiveCocoa/DynamicProperty.swift

```diff
@@ -37,7 +37,13 @@ public final class DynamicProperty<Value>: MutablePropertyProtocol {
 	///              Most UI controls are not!
 	public var producer: SignalProducer<Value?, NoError> {
 		return (object.map { $0.reactive.values(forKeyPath: keyPath) } ?? .empty)
-			.map { $0 as! Value }
```

Comment:
@sharplet There is an existing case though. https://github.com/ReactiveCocoa/ReactiveCocoa/pull/3313#issuecomment-262998365